### DISCUSSION TITLE
What's so special about the Microsoft builds of OpenJDK?

### DISCUSSION THREAD

**Discussion by ksze** [microsoft/openjdk#158] [Q&A] [answered]
Why would somebody want to use the Microsoft builds instead of, say, one of these:
- https://jdk.java.net/17/
- https://adoptopenjdk.net/

I know that you don't have performance benchmark comparisons yet, according to another discussion thread. But there must be at least some theoretical selling points. What are they? They are not mentioned on https://docs.microsoft.com/en-gb/java/openjdk/overview, unless you mean your real selling point is the inclusion of some backported fixes.

**Comment by karianna** [comment, marked as answer]
Hi @ksze - As it stands today the benefits are primarily around our support on Azure and the ports to Windows ARM and Apple M1 (we authored a chunk of both of those) + some Md5sum enhancements.

Going forwards we'll be adding performance enhancements, especially for cloud-based / container workloads (no formal roadmap yet, but as part of that we will be releasing benchmarks, etc).
